integration-tests/gatsby-source-wordpress/test-fns/data-resolution.js

```diff
@@ -89,7 +89,7 @@ describe(`data resolution`, () => {
     expect(gatsbyResult.data.allWpContentNode.nodes.length).toBe(
       // we add a media item node before running our warm cache build.
       // so 30 before 31 after
-      isWarmCache ? 30 : 29
+      isWarmCache ? 31 : 30
```

Comment:
as far as I can tell this is because not all plugins were being activated before (this PR fixes that)